applications/nrf5340_audio/README.rst

```diff
@@ -778,6 +775,16 @@ Selecting the BIS mode
 The CIS mode is the default operating mode for the application.
 You can switch to the BIS mode by adding the ``CONFIG_TRANSPORT_BIS`` Kconfig option set to ``y``  to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
 
+.. _nrf53_audio_app_configuration_select_bidirectional:
+
+Selecting the CIS bidirectional communication
+======================
```

Comment:
Please align the number of = with the length of the header to fix the doc build.